### PR TITLE
Added test create New View with existing name

### PR DESCRIPTION
**Pull request by olpolezhaeva**

### CODE REVIEW

**Comment by olpolezhaeva** [comment]
[AT_026.003 |My Views > Create new view > Check create new view with an existing name](https://trello.com/c/QdMykWkc/196-at026003-my-views-create-new-view-check-create-new-view-with-an-existing-name)

File: src/test/java/CreateNewViewWithExistingName.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+public class CreateNewViewWithExistingName extends BaseTest {
+
+    @Test
+    public void testCreateNewViewWithAnExistingName() {
+        getDriver().findElement(By.xpath("//a[@title='New View']")).click();
+        String nameOfView = "My new view";
+        getDriver().findElement(By.id("name")).sendKeys(nameOfView);
+        getDriver().findElement(By.xpath("//label[text()='My View']")).click();
+        String errorMessageExpected = getDriver().findElement(
```
Comment (GitForProjects) [inline src/test/java/CreateNewViewWithExistingName.java:14]: here you mixed up the names of the variables
line 14 - it's Actual message (то, что мы получили на странице)
line 16 - Expected message (то, что мы одидаем получить)

Comment (olpolezhaeva) [reply src/test/java/CreateNewViewWithExistingName.java]: O, yes, fixed

**Review by GitForProjects — APPROVED** [review]
looks good

File: src/test/java/CreateNewViewWithExistingName.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class CreateNewViewWithExistingName extends BaseTest {
+    @Test
+    public void testCreateNewViewWithAnExistingName() {
+        getDriver().findElement(By.xpath("//a[@title='New View']")).click();
+        String nameOfView = "My new view";
```
Comment (azellus-tercia) [inline src/test/java/CreateNewViewWithExistingName.java:10]: You can use this string as a constant outside of a method.

**Review by azellus-tercia — APPROVED** [review]

**Pull request merged**